Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -176,6 +191,14 @@ extension BrowserViewController: TabManagerDelegate {
         }
     }
 
+    func selectedIndex() -> Int {
+        return tabManager.selectedIndex
+    }
+
+    func selectedTab() -> Browser? {
+        return tabManager.selectedTab?
+    }
```

Comment:
Not sure why you created these, but I'd remove them. TabTrayController already holds a tabManager.